tests/resources/pr_1783_labels.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "id": 107669776,
+    "url": "https://api.github.com/repos/dimagi/commcare-android/labels/bug",
+    "name": "bug",
+    "color": "fc2929",
+    "default": true
+  },
+  {
+    "id": 116252758,
+    "url": "https://api.github.com/repos/dimagi/commcare-android/labels/ready%20for%20review",
+    "name": "ready for review",
+    "color": "fbca04",
+    "default": false
+  }
+]
```

Comment:
nit: newlines missing

Reply:
Ah sorry, will add. I'm curious why this matters; it's not something we've ever worried about in mobile repos before, although maybe it should be

Reply:
Not a real reason, but you can read some comments about it here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
The major benefit I see is that if you are adding new things at the end of the file the diff will look "correct", rather than show that you also modified the last line in the file. This is the same reason we use dangling commas in python and javascript. 
Also, a lot of text editors will add the newline at the end on save, which will then show up in future commits.